javascriptv3/example_code/reactnative/ReactNativeApp/android/app/build.gradle

```diff
@@ -0,0 +1,170 @@
+apply plugin: "com.android.application"
+apply plugin: "com.facebook.react"
+
+import com.android.build.OutputFile
+
+/**
+ * This is the configuration block to customize your React Native Android app.
+ * By default you don't need to apply any configuration, just uncomment the lines you need.
+ */
+react {
+    /* Folders */
```

Comment:
These are the default comments, no? Are they useful to keep?

Reply:
I find utility in keeping it as close to the generated react-native code as possible. It's a getting started in react guide that explicitly says to run `react-native init`.